yb-voyager/cmd/common.go

```diff
@@ -247,3 +250,69 @@ func nameContainsCapitalLetter(name string) bool {
 	}
 	return false
 }
+
+// This function is invoked at the end of export schema to process files containing statments of the type `\i <filename>.sql`, merging them together.
+func processImportDirectives(fileName string) error {
+	if _, err := os.Stat(fileName); os.IsNotExist(err) {
+		return nil
+	}
+	// Create a temporary file after appending .tmp extension to the fileName.
+	tmpFileName := fileName + ".tmp"
+	tmpFile, err := os.Create(tmpFileName)
+	if err != nil {
+		return fmt.Errorf("error while creating temp file %s to store processed DDLs: %v", tmpFileName, err)
```

Comment:
Be precise and concise in error messages.
As a context, you have to tell the action the code was trying to perform when the error occurred.
`"error while"` is redundant in an `error` object.
I would write the statement as:
```
return fmt.Errorf("create %q: %w", tmpFileName, err)
```
Note the `%w` format specifier for `err` objects.